diff --git a/binding.gyp b/binding.gyp
index db98bb83b50ecc75aea7ccddd9a20c99d3494c13..b6ad18936d56c2441fac9375c5630290f402ad6b 100644
--- a/binding.gyp
+++ b/binding.gyp
@@ -32,7 +32,7 @@
         'src/thread_name.cc'
       ],
       'cflags_cc': [
-        '-std=c++14'
+        '-std=c++17'
       ],
       'defines': [
         '_FILE_OFFSET_BITS=64',
@@ -60,7 +60,7 @@
           ['OS=="mac"', {
             'xcode_settings': {
               'OTHER_CFLAGS': [
-                '-std=c++1y',
+                '-std=c++17',
                 '-stdlib=libc++',
                 '-arch x86_64',
                 '-arch arm64'
